modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/jaxrs/deserializer/JSONStringStdDeserializer.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.jaxrs.deserializer;
```

Comment:
I think this package is not correct. This is not related with JAX-RS, but with Jackson. There is already a class that implements a Jackson interface (VulcanPropertyFilter). Please, move to the proper package. Remember that in Liferay we usually use a similar package that the interface that is implemented belongs to.

For example:

```
package com.fasterxml.jackson.databind.ser;

[...]

public interface PropertyFilter {
```

```
package com.liferay.portal.vulcan.jackson.databind.ser;

[...]
public class VulcanPropertyFilter
	extends SimpleBeanPropertyFilter implements PropertyFilter {
```

Reply:
You are right @ccorreagg!! I will do it but instead of ser, use the deser

Reply:
However, I've seen that all the serializers are in jax-rs packages, from portal-vulcan-impl. That's why I moved that into that package

Reply:
What do think?